src/99-appendices/03-hed.md

```diff
@@ -185,3 +185,41 @@ any present HED information will be validated using the latest version of the HE
 which is bound to result in problems.
 Hence, it is strongly RECOMMENDED that the `HEDVersion` field be included when using HED
 in a BIDS dataset.
+
+### Using HED library schemas
+
+HED also allows you to use one or more specialized vocabularies along with
+the base vocabulary. These specialized vocabularies are developed by
+communities of users and are available in the GitHub
+[hed-schema-library](https://github.com/hed-standard/hed-schema-library) repository.
+
+Example: The following `dataset_description.json` file specifies that the
+[`HED8.1.0.xml`](https://github.com/hed-standard/hed-specification/tree/master/hedxml/HED8.1.0.xml)
+base schema should be used along with the
+SCORE library for clinical neurological annotation and a test library
+located at [HED_score_0.0.1.xml](https://github.com/hed-standard/hed-schema-library/blob/main/library_schemas/score/hedxml/HED_score_0.0.1.xml) and [HED_testlib_1.0.2.xml](https://github.com/hed-standard/hed-schema-library/blob/main/library_schemas/testlib/hedxml/HED_testlib_1.0.2.xml), respectively.
+
+```JSON
+{
+  "Name": "A great experiment",
+  "BIDSVersion": "1.7.0",
+  "HEDVersion": {
+      "base": "8.1.0",
+      "libraries": {
+          "sc": "score_0.0.1",
+          "ts": "testlib_1.0.2"
+      }
+  }
+}
+```
+The `sc:` and `ts:` are user-chosen prefixes used to distinguish the sources
+of the terms in the HED annotation.
+These prefixes must be alphanumeric.
```

Comment:
can they start with a number, e.g. be "1"?

Reply:
Yes they can start with a number.

Reply:
gut feeling tells me it might cause us pains, but I will just go take some Pepto to treat it ;)